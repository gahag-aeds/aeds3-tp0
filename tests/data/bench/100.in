100 50
49 59 3 12 45 51 12 40 84 88 33 52 94 0 38 1 88 48 52 99 2 56 77 34 88 14 16 61 74 91 27 75 2 30 40 99 33 4 92 18 92 25 22 39 77 61 92 17 9 44 16 63 1 45 97 89 59 65 3 86 56 30 61 10 13 53 10 46 58 2 16 2 79 39 41 56 52 33 25 61 78 42 24 31 87 73 20 99 38 75 85 47 58 98 9 71 52 71 17 62 
Sub 65 97
Sum 24 56
Add 43 75
Min 9 41
Sub 59 91
Min 50 82
Add 11 43
Max 45 77
Sub 24 56
Min 53 85
Add 38 70
Max 2 34
Add 49 81
Min 63 95
Add 43 75
Sum 62 94
Sub 12 44
Sum 50 82
Add 40 72
Sum 27 59
Add 66 98
Min 62 94
Add 63 95
Sum 35 67
Add 18 50
Max 48 80
Sub 47 79
Max 1 33
Sub 28 60
Min 28 60
Add 6 38
Max 43 75
Sub 1 33
Sum 48 80
Sub 51 83
Min 9 41
Add 47 79
Min 62 94
Sub 53 85
Min 21 53
Add 64 96
Max 23 55
Sub 22 54
Max 62 94
Sub 35 67
Sum 65 97
Sub 6 38
Max 63 95
Sub 64 96
Max 46 78
