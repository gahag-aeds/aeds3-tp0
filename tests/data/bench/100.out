1587
0
1
98
1
100
2
1539
1487
1633
5
1620
101
100
1
99
1417
0
5
1
99
100
1618
100
97
